README.MD

```diff
@@ -1,3 +1,6 @@
 # Pandit.AI
 - Dictates pooja steps
     - It waits for the feedback before dictating next step
+- It will use Hindi language to dicatate pooja steps
+- It will listen for words such as "ho gaya" to move to next step of pooja
+- It will also have ability to play bhakti geet
```

Comment:
Please use "song" instead of  "geet".

Reply:
I think this still looks good. I would say lets stick with geet only

Reply:
okay